src/main/java/org/reaktivity/nukleus/http/internal/util/HttpUtil.java

```diff
@@ -101,7 +101,7 @@ public static boolean isPathValid(
         if (valid)
         {
             byte_loop:
-            for (int index = (originalCapacity < 8) ? 0 : capacity; capacity > 0; index++, capacity--)
+            for (int index = (originalCapacity <= 8) ? 0 : capacity; capacity > 0; index++, capacity--)
```

Comment:
If we make this change, then it works, but it forces the `8 byte` ascii string to use the less efficient per-byte approach instead of the 8-byte stride in the main loop above.

If we revert this change, but instead change main loop to be `capacity >= Long.BYTES` then we get the benefit of the 8-byte striding approach for `8 byte` ascii strings, agree?

Reply:
Yeah much better.